shared/terraform.go

```diff
@@ -9,21 +9,32 @@ import (
 	"testing"
 
 	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/rancher/distros-test-framework/config"
 )
 
-func addTerraformOptions(product string) (*terraform.Options, string, error) {
+func setTerraformOptions() (*terraform.Options, string, error) {
+	cfg, err := config.AddEnv()
```

Comment:
you dont need here add env , since we are add on suite.
and on your `setTerraformOptions` you can directly call 
```
	product := os.Getenv("ENV_PRODUCT")
	airgap := os.Getenv("ENV_MODULE")

```

Reply:
Updated